### ISSUE TITLE
[readme][dual-boot] Add support for SteamOS 3

### ISSUE THREAD

**Issue by LukeShortCloud** [LukeShortCloud/winesapOS#939]
Test and document how to dual-boot with SteamOS 3.

**Comment by LukeShortCloud** [comment]
The partitioning starts around the 5 minute mark of the video below. They used GParted in the SteamOS recovery live environment to shrink the last partition.

https://youtu.be/IraA82T1bxE?si=lpWd5alpTTqOlBQU

**Comment by LukeShortCloud** [comment]
Given that you can run winesapOS off of a SD card on a SteamOS handheld, I feel like this is much lower priority.

**Comment by LukeShortCloud** [comment]
SteamOS 3.8.10 was just released as the first stable release of 3.8.

https://steamcommunity.com/games/1675200/announcements/detail/697641379212298073

**Comment by LukeShortCloud** [comment]
Here is the first installer for the stable 3.8 series (based on 3.8.11): https://steamdeck-images.steamos.cloud/steamdeck/20260620.1/steamdeck-20260620.1-3.8.11.img.zst

**Comment by LukeShortCloud** [comment]
Based on a fresh install of SteamOS 3.8.10, it appears to be exactly the same as dual-booting with Ubuntu. There is an existing EFI partition but it is way too small and has other Steam binaries. A new EFI partition is recommended for dual-booting other Linux distributions alongside winesapOS anyways.

I pushed up documentation but now I need to actually test it.

**Comment by LukeShortCloud** [comment]
Before testing dual-boot with SteamOS, we should have the OGC kernel installed.

Blocked-by: https://github.com/winesapOS/winesapOS/issues/1179

**Comment by LukeShortCloud** [comment]
This project can help re-use the existing EFI partition (which probably needs to be expanded):

https://github.com/jlobue10/SteamDeck_rEFInd

**Comment by LukeShortCloud** [comment]
Success, it works! I just need to make some minor changes to the documentation for both SteamSO and dual-boot in general.

However, the bad news is that I found a lot of bugs with the dual-boot installer.
